src/config/constants/addresses.ts

```diff
@@ -724,4 +724,68 @@ export const getGhTokenListLogoUrl = (chainId: number, address: string) =>
 // oasis: 42262,
 
 export const UNSUPPORTED_WAGMI_CHAIN = [122, 128, 106, 42262];
+
+export const SUPPORTED_CHAINS_BY_CONNEXT: Record<number, { domainId: string; name: string; network: string }> = {
```

Comment:
You could extend de NetworkStruct type to add the domainId, and with that you would already have if it is mainnet or a testnet, plus all the names. Furthermore, i would assign keys here as `[NETWORKS.optimism.chainId]`